gcalcli/gcalcli.py

```diff
@@ -1246,7 +1293,16 @@ def QuickAddEvent(self, event_text, reminders=None):
 
         return new_event
 
-    def AddEvent(self, title, where, start, end, descr, who, reminders):
+    def AddEvent(
```

Comment:
Same nit as above about the style, be consistent with the whitespace handling you see in the other function signatures (see ModifyEvent, below).